ranlab/.nvmrc

```diff
@@ -0,0 +1 @@
+12.9.1
```

Comment:
@whymarrh For some reason yarn kept shitting the bed @13.6.0... I have no idea why. Thoughts?

Reply:
I don't know why but the even numbers are the good versions, and tools usually only test against the even numbers so this is fine

Reply:
That makes little sense, but okay let's keep it 😂